src/bundle/Resources/views/content/content_edit/user_edit.html.twig

```diff
@@ -48,3 +48,23 @@
 {% block close_button %}
     <a class="ez-content-edit-container__close" href="{{ path('_ezpublishLocation', {'locationId': user.versionInfo.contentInfo.mainLocationId}) }}"></a>
 {% endblock %}
+
+{% block form_before %}
+    {{ ezplatform_admin_ui_component_group('content-edit-form-before', {
+        'content': user,
+        'contentType': contentType,
+        'location': location,
+        'parent_location': user.versionInfo.contentInfo.mainLocationId,
```

Comment:
I'm not sure if this should be `mainLocationId` I think in content create the real parentLocationId is passed.
//cc @webhdx

Reply:
in content edit, whole `parent_location` object is passed. Not sure how this works then.

Also, passing `mainLocationId` could cause bug with missing privilege rights to access it, if I am right. (@mikadamczyk )